portal-api/src/v1/routes.js

```diff
@@ -58,15 +59,16 @@ openRouter
   .post(
     checkApiKey,
     param('userId').isMongoId().withMessage('Value must be a valid MongoId'),
-    param('signInToken').isHexadecimal().withMessage('Value must be a hexadecimal string'),
+    param('signInToken')
```

Comment:
We need to make a decision on where we are validating input -- I'm quite happy to keep it here as it'll mean we can remove a lot of boilerplate checking these are valid inside controllers/services